int64 joint_number
float32 degree
---
bool success